db/fixtures/miq_policies.yml

```diff
@@ -0,0 +1,29 @@
+---
+- :name: analyze_new_images
+  :description: Analyze all new images
+  :expression:
+  :towhat: ContainerImage
+  :guid:
+  :created_by: admin
+  :updated_by: admin
+  :notes:
+  :active: true
+  :mode: control
+  :conditions: []
+- :name: handle_high_severity_failures
+  :description: Prevent images with high severity cves from running at openshift
```

Comment:
Same here...since it's container specfic it should say "container" somewhere in the name and description.  Additionally, CVE should be caps (probably like "CVEs" in this case), and OpenShift is a product name so it should be properly capitalized as OpenShift.  You *might* have to make it fully spelled out as "Red Hat OpenShift" or "OpenShift Enterprise" unless it's not really Enterprise specific (you get the idea).